tests/python/gaia-ui-tests/gaiatest/tests/graphics/test_marketplace_download_execute_map_app.py

```diff
@@ -34,62 +34,48 @@ def test_install_and_execute_bing_map(self):
 
         results = marketplace.search(self.app_search)
         first_result = results.search_results[0]
-        app_name = first_result.get_app_name()
         first_result.tap_install_button()
 
-        # Confirm the installation and wait for the app icon to be present
+        # Confirm the installation
```

Comment:
If you'd ditch the comment completely, I don't think anything of value would be lost.